File: src/main/kotlin/com/youtrack/ui/IssueList.java
package com.youtrack.ui;

import com.youtrack.issue.Issue;

import javax.swing.*;

public class IssueList extends JList<Issue> {

    public IssueList(DefaultListModel<Issue> model) {
        super(model);
        setCellRenderer(new MyListCellRenderer());
    }

    public IssueList() {
        setCellRenderer(new MyListCellRenderer());
    }
}

File: src/main/kotlin/com/youtrack/ui/IssueList.kt
package com.youtrack.ui

import com.youtrack.issue.Issue
import javax.swing.DefaultListModel
import javax.swing.JList

class IssueList(model: DefaultListModel<Issue>) : JList<Issue>(model) {

    init {
        cellRenderer = MyListCellRenderer()
    }

    constructor() : this(DefaultListModel())
}